src/mixins/common-mixin.ts

```diff
@@ -40,6 +40,12 @@ export class CommonMixin extends Vue {
       case EntityTypes.SP: return 'BC Sole Proprietorship'
       case EntityTypes.UL: return 'BC Unlimited Liability Company'
 
+      // Continuation In Entity Types:
+      case EntityTypes.C: return 'Continuation In (BC Limited Company)'
+      case EntityTypes.CBEN: return 'Continuation In (Benefit Company)'
+      case EntityTypes.CCC: return 'Continuation In (BC Community Contribution Company)'
+      case EntityTypes.CUL: return 'Continuation In (BC Unlimited Liability Company)'
```

Comment:
To be honest, I think these cases don't get used, because as far as Namerequest UI is concerned, the subject NR is for a "BC Benefit Company" (for example), not a CBEN. I think only LEAR cares that this is a continued in benefit company.